test/integration/suites/k8s-sigstore-e2e/00-setup

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Create a temporary path that will be added to the PATH to avoid picking up
+# binaries from the environment that aren't a version match.
+mkdir -p ./bin
+
+KIND_PATH=./bin/kind
+KUBECTL_PATH=./bin/kubectl
+
+# Download kind at the expected version at the given path.
+download-kind "${KIND_PATH}"
+
+# Download kubectl at the expected version.
+download-kubectl "${KUBECTL_PATH}"
+
+# We must supply an absolute path to the configuration directory. Replace the
+# CONFDIR variable in the kind configuration with the conf directory of the 
+# running test.
+sed -i.bak "s#CONFDIR#${PWD}/conf#g" conf/kind-config.yaml
+rm conf/kind-config.yaml.bak
+
+#create folder which will contain the registry certificate information
```

Comment:
can you add an space after `#` in all comments? and to be consistent first word must start in uppercase